device/bgp/evpn/bgpevpnevivxlan.yaml

```diff
@@ -0,0 +1,15 @@
+components:
+  schemas:
+    Bgp.EvpnEviVxlan:
+      x-include:
+      - './bgpevpnevibase.yaml#/components/schemas/Bgp.EvpnEviBase'
+      type: object
+      properties:        
+        replication_type:
```

Comment:
currently only ingress replication supported so need to add now,maybe later when other variants come